packages/calcite-components/src/components/combobox/combobox.stories.ts

```diff
@@ -333,21 +333,56 @@ export const nestedItems = (): string => html`
   </div>
 `;
 
-export const longItems_TestOnly = (): string =>
-  html`<style>
-      calcite-combobox {
-        width: 260px;
-      }
-      calcite-combobox-item {
-        width: 260px;
-      }
-    </style>
-    <calcite-combobox open>
-      <calcite-combobox-item text-label="Layers">
+const style = html`
+  <style>
+    calcite-combobox {
+      width: 260px;
+    }
+    calcite-combobox-item {
+      width: 260px;
+    }
+  </style>
+`;
+
+export const longItemsAllSelectionModes = (): string => html`
+  ${style}
+  <div style="display: flex; flex-direction: column;"> 
```

Comment:
These render on top of one another in the Chromatic screenshots, can we separate the with css or into different stories?
<img width="835" alt="Screenshot 2024-02-18 at 2 16 57 PM" src="https://github.com/Esri/calcite-design-system/assets/4733155/86770b62-0d8f-4336-bdc9-90d63f743d06">